DigitalLearningSolutions.Web.Tests/Helpers/SelectListHelperTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Helpers
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Web.Helpers;
+    using FluentAssertions;
+    using FluentAssertions.Execution;
+    using NUnit.Framework;
+
+    public class SelectListHelperTests
+    {
+        private const string item1 = "item1";
+        private const string item2 = "item2";
+
+        [Test]
+        public void GetOptionsWithSelectedText_returns_populated_list()
+        {
+            // Given
+            var inputList = new List<(int id, string name)>
+            {
+                (1, item1),
+                (2, item2)
+            };
+
+            // When
+            var result = SelectListHelper.GetOptionsWithSelectedText(inputList, item1);
+
+            using (new AssertionScope())
+            {
+                result.Should().NotBeNullOrEmpty();
+                result.Count().Should().Be(2);
+
+                result.SingleOrDefault(i => i.Value == 1.ToString()).Should().NotBeNull();
+                result.SingleOrDefault(i => i.Value == 1.ToString()).Selected.Should().BeTrue();
+

```

Comment:
I'm quite keen on minimising logic in tests so I wonder if we could do something like this instead:
```
result.ElementAt(0).Value.Should().Be("1"); 
result.ElementAt(0).Text.Should().Be("item1"); 
etc
```